docusaurus.config.js

```diff
@@ -135,146 +135,164 @@ const config = {
           },
         ],
       },
-      footer: {
-        style: 'dark',
-        links: [
+footer: {
```

Comment:
please fix the indentation

Reply:
Okay

Reply:
Can you want change footer or not I don't changed it only change navbar for responsive size /  device but I don't have more links for your footer so, i don't to try for footer but navbar fully working for responsive devices.